pruner/pruner.go

```diff
@@ -0,0 +1,146 @@
+package pruner
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/docker/distribution/configuration"
+	"github.com/docker/distribution/context"
+	"github.com/docker/distribution/uuid"
+	"github.com/docker/distribution/version"
+	"github.com/spf13/cobra"
+)
+
+const (
+	binaryName      = "pruner"
+	helpMessageTmpl = `%s allows to print and/or delete registry's orphaned blobs.
+
+Make sure that registry instance isn't running or is running in read-only mode
+before launching this executable.`
+)
+
+// Cmd is a cobra command for running the registry.
+var Cmd = &cobra.Command{
```

Comment:
Move command definition into `registry` package, since this is where we are defining cli endpoints.